packages/data/src/registry.js

```diff
@@ -198,6 +202,18 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 		config.subscribe( globalListener );
 	}
 
+	/**
+	 * Registers a new store.
+	 *
+	 * @param {import('./types').WPDataStoreDefinition} storeDefinition Store definition.
+	 */
+	function register( storeDefinition ) {
+		registerGenericStore(
+			storeDefinition.name,
+			storeDefinition.__internalConfig
```

Comment:
This needs to be `__internalConfig( registry )`, otherwise everything will fail.

Reply:
yes, I missed this, fixed now.